docs/interfaces/_types_.handler.html

```diff
@@ -103,7 +103,7 @@ <h3 class="tsd-before-signature">Callable</h3>
 					<li class="tsd-description">
 						<aside class="tsd-sources">
 							<ul>
-								<li>Defined in <a href="https://github.com/asos-craigmorten/opine/blob/8956e50/src/types.ts#L53">types.ts:53</a></li>
+								<li>Defined in <a href="https://github.com/FaberVitale/opine/blob/89ef311/src/types.ts#L53">types.ts:53</a></li>
```

Comment:
Hmm, will be a bit odd having links out from one repo to another... though atm these aren't even visible as gh doesn't support non `master` branches for gh pages.

I think rather than burdening myself and other contributors with generating docs every PR, I might look into creating a gh action to do this for us on a post merge hook.

Reply:
https://github.com/asos-craigmorten/opine/issues/29